packages/selenium/src/AxeBuilder.cs

```diff
@@ -57,6 +71,29 @@ public AxeBuilder(IWebDriver webDriver, AxeBuilderOptions options)
             _AxeBuilderOptions = options;
         }
 
+        /// <summary>
+        /// Use frameMessenger with &lt;same_origin_only&lt;.
+        /// This disables use of axe.runPartial() which is called in each frame, and
+        /// axe.finishRun() which is called in a blank page. This uses axe.run() instead,
+        /// but with the restriction that cross-origin frames will not be tested.
+        /// </summary>
+        /// <param name="legacyMode"></param>
+        [Obsolete("Legacy Mode is being removed in the future. Use with caution!")]
+        public AxeBuilder UseLegacyMode(bool legacyMode = false)

```

Comment:
It would be good to update the `README.md` API docs to include the 2 new public `AxeBuilder` methods.